README.md

```diff
@@ -25,26 +33,30 @@ With this project, Aras has a few federated object which are querying Gitlab thr
 1. Backup your database and store the BAK file in a safe place.
 2. Open up the Aras Package Import tool.
 3. Enter your login credentials and click **Login**
-  * _Note: You must login as root for the package import to succeed!_
+		* _Note: You must login as root for the package import to succeed!_
 4. Enter the package name in the TargetRelease field.
   * Optional: Enter a description in the Description field.
 5. Enter the path to your local `..\PLM-ALM-with-GitLab\Import\imports.mf` file in the Manifest File field.
-6. Select **aras.labs.plmalmwithgitlab** in the Available for Import field.
+6. Select **aras.labs.gitlab** in the Available for Import field.
```

Comment:
Why change the name of the package? This will cause issues for anyone who is installing over an existing version of the project.

Reply:
This was a typo on the part of the originator. I am correcting the package name in the documentation. .mf file has not been changed.

Reply:
👍